packages/ckeditor5-ui/tests/button/buttonview.js

```diff
@@ -298,21 +298,17 @@ describe( 'ButtonView', () => {
 				expect( view.element.hasAttribute( 'aria-pressed' ) ).to.be.false;
 			} );
 
-			it( '-checked reacts on #isOn', () => {
-				view.isOn = true;
-				expect( view.element.attributes[ 'aria-checked' ].value ).to.equal( 'true' );
-
-				view.isOn = false;
-				expect( view.element.hasAttribute( 'aria-checked' ) ).to.be.false;
-			} );
-
 			it( '-label reacts on #ariaLabel', () => {
 				view.ariaLabel = undefined;
 				expect( view.element.hasAttribute( 'aria-label' ) ).to.be.false;
 
 				view.ariaLabel = 'Foo';
 				expect( view.element.attributes[ 'aria-label' ].value ).to.equal( 'Foo' );
 			} );
+
+			it( '-checked is not present', () => {
+				expect( view.element.hasAttribute( 'aria-checked' ) ).to.be.false;
```

Comment:
This test doesn't check against regression. The reason is: it was already `false` when `view.isOn` was `false`. And it is by default. Probably there is missing `view.isOn = true`.